src/utils/db.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', 'admin', {
```

Comment:
i glad that you so trust us but would be nice to store this creds in .env file and use just variable here)